src/components/AppComponents/TimeoutPopup/index.tsx

```diff
@@ -87,9 +96,7 @@ export default function TimeoutPopup(props) {
     if (timeoutState.timeRemaining < 60 && timeoutState.timeRemaining % 20 === 0) {
       dispatch({
         type: 'UPDATE_SCREEN_READER_COUNTDOWN',
-        payload: `${t('FOR_YOUR_SECURITY_WE_WILL_SIGN_YOU_OUT')} ${timeoutState.timeRemaining} ${t(
-          'SECONDS'
-        )}`
+        payload: `${screenReaderContentDisplay()} ${timeoutState.timeRemaining} ${t('SECONDS')}`
```

Comment:
timeoutText() function can be used here

Reply:
for screenreader we have a different logic, it should read it after every 20 secs once it's past 1 min